src/main/java/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As I can see, your log4j2.xml is located under java folder. Have you tested your solution and did your logging really work?